client/flutter/lib/generated/intl/messages_ar.dart

```diff
@@ -32,6 +32,10 @@ class MessageLookup extends MessageLookupByLibrary {
     "homePagePageButtonWHOMythBusters" : MessageLookupByLibrary.simpleMessage("هوو أسطورة-بوسترز"),
     "homePagePageButtonWHOMythBustersDescription" : MessageLookupByLibrary.simpleMessage("تعرف على الحقائق المتعلقة بفيروس كورونا وكيفية منع انتشاره"),
     "homePagePageButtonYourQuestionsAnswered" : MessageLookupByLibrary.simpleMessage("الإجابة على أسئلةك"),
+    "homePagePageSliverListSettingsHeader1": MessageLookupByLibrary.simpleMessage("جمع البيانات"),
```

Comment:
@sapte91 how are we tracking which translations here are MT vs confirmed? do we need any additional process when checking in these types of changes, and/or should they not be included in these types of PRs? will defer to you here

Reply:
great callout. for now, we're planning on overwriting MT files but this will definitely need to be addressed for future releases! so we're thinking about this and will have an update to share shortly after v1.